Procfile.dev

```diff
@@ -1,5 +1,5 @@
 # You can run these commands in separate shells
-web: rails s -p 3000
+web: rails s
```

Comment:
**Question:** Removed the port because of Discourse local env development issues?

Reply:
Foreman uses $PORT environment variable to set the port. If you hardcode the port then you can change it. For example both discourse and this app run in port 3000.